pilot/test/util/kubernetes.go

```diff
@@ -59,7 +59,26 @@ func CreateNamespaceWithPrefix(cl kubernetes.Interface, prefix string, inject bo
 	log.Infof("Created namespace %s", ns.Name)
 	return ns.Name, nil
 }
-
+// CreateNamespaceWithPrefix creates a fresh namespace with the given prefix
+func CreateNamedNamespace(cl kubernetes.Interface, namespace string, inject bool) (string, error) {
```

Comment:
CreateNamespace instead? All the namespace has a name.

I see that there is only one line difference between CreateNameNamespace and CreateNamespaceWithPrefix